addons/website_event_exhibitor/static/src/components/exhibitor_connect_closed_dialog/exhibitor_connect_closed_dialog.js

```diff
@@ -25,6 +26,13 @@ export class ExhibitorConnectClosedDialog extends Component {
         sponsorData.website_description = sponsorData.website_description || "";
         sponsorData.website_description = markup(sponsorData.website_description);
         this.formatEventStartRemaining = formatDuration(sponsorData.event_start_remaining, true);
+        this.formatEventDateBegin = formatDateTime(
+            deserializeDateTime(sponsorData.event_date_begin),
+            {
+                tz: sponsorData.event_date_tz,
+                format: "MMM d, yyyy, h:mm a"
```

Comment:
Would it be possible to use builtins like "DATE_MED" etc..? The annoying thing with format strings is that they don't adapt to locale. Where in most of the world "MMM d" isn't really expected.

If not I guess it's fine but this seems like the only place we use this formatting (?)